content/articles/files-and-exeptions-in-python/index.md

```diff
@@ -0,0 +1,265 @@
+ 
```

Comment:
The file directory name must be `files-and-exceptions-in-python`

`exeptions` -> `exceptions`